src/main/python/pybuilder/errors.py

```diff
@@ -56,12 +54,14 @@ def __init__(self, first, second=None):
                 CircularTaskDependencyException, self).__init__("Circular task dependency detected between %s and %s",
                                                                 first,
                                                                 second)
+        else:
+            super(CircularTaskDependencyException, self).__init__(first)
```

Comment:
By not calling an init we don't call a super into exceptions. This makes the exception unpicklable.